### PR TITLE
Label:Refactoring related functions of color.

### PR DESCRIPTION
**Pull request by WenhaiLin**
Adds `Label::setTextColor`, text color will not affect shadow & stroke color.
But `Label::setColor` would blend all the whole texture.
Use case of changing color in `cpp-tests` has been changed `setTextColor`.

### CODE REVIEW

File: cocos/2d/CCGLProgram.cpp [hunk 1]
```diff
@@ -58,8 +58,9 @@ of this software and associated documentation files (the "Software"), to deal
 const char* GLProgram::SHADER_NAME_POSITION_U_COLOR = "ShaderPosition_uColor";
 const char* GLProgram::SHADER_NAME_POSITION_LENGTH_TEXTURE_COLOR = "ShaderPositionLengthTextureColor";
 
-const char* GLProgram::SHADER_NAME_LABEL_DISTANCEFIELD_NORMAL = "ShaderLabelNormol";
-const char* GLProgram::SHADER_NAME_LABEL_DISTANCEFIELD_GLOW = "ShaderLabelGlow";
+const char* GLProgram::SHADER_NAME_LABEL_DISTANCEFIELD_NORMAL = "ShaderLabelDFNormol";
```
Comment (dumganhar) [inline cocos/2d/CCGLProgram.cpp]: `Normol` -> `Normal`?

**Comment by HoGarfield** [comment]
How to fix the issue in version 2.x

**Comment by dumganhar** [comment]
@HoGarfield 
`New label` wasn't supported in `2.x` version.

**Comment by HoGarfield** [comment]
那请问我应该怎么去修复字体描边的问题，在版本2.x中                             原始邮件                             发件人: James Chennotifications@github.com收件人: cocos2d/cocos2d-xcocos2d-x@noreply.github.com抄送: garfield_hogarfield_ho@qq.com发送时间: 2014年3月29日(周六) 17:35主题: Re: [cocos2d-x] Label:Refactoring related functions of color. (#5893)@HoGarfield New label wasn't supported in 2.x version.

**Pull request merged**